src/main/kotlin/no/nav/syfo/utils/VarselUtil.kt

```diff
@@ -8,12 +8,16 @@ import no.nav.syfo.db.fetchSykmeldingerIdByPlanlagtVarselsUUID
 import no.nav.syfo.kafka.consumers.oppfolgingstilfelle.domain.Oppfolgingstilfelle39Uker
 import no.nav.syfo.kafka.consumers.oppfolgingstilfelle.domain.Syketilfelledag
 import no.nav.syfo.syketilfelle.domain.Tag
+import java.time.DayOfWeek
 import java.time.LocalDate
 import java.time.temporal.ChronoUnit
 
 val antallUker39UkersVarsel = 39L
 val antallDager39UkersVarsel = antallUker39UkersVarsel * 7L + 1
 val SVAR_MOTEBEHOV_DAGER: Long = 112
+val MAX_ANTALL_REMAINING_SYKEDAGER_39_UKERS_VARSEL = 65
```

Comment:
Hm, kanskje gå fra blanding av norsk og engelsk til kun engelsk? Kanskje noe ala

REMAINING_SYKEDAGER_UNTIL_39_UKERS_VARSEL
REMAINING_WEEKS_UNTIL_39_UKERS_VARSEL

eller noe? For å få fagbegreper på norsk, men resten på engelsk.

Reply:
Enig, fikset.